pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -361,6 +304,30 @@ func (r *ReconcileHyperConverged) doReconcile(req *common.HcoRequest) (reconcile
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileHyperConverged) constructHcoRequest(ctx context.Context, request reconcile.Request) (*common.HcoRequest, error) {
```

Comment:
This function does two things. It check if the source of the request is a change in HC or in a secondery watched resource, *and* creates the hcoRequest. The latest already implemented in common.NewHcoRequest.

I think this method need a new name and the last line should be placed in the original place, out of this method.

Reply:
Done.